sql/hive/pom.xml

```diff
@@ -36,6 +36,12 @@
   </properties>
 
   <dependencies>
+    <!-- Added for sbt issue, expected to be evicted -->
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-library</artifactId>
+      <version>2.10.3</version>
```

Comment:
Hm, I see. That feels error-prone or like there must be a better answer; I don't know of one though so can't strongly object.